src/text-editor-component.coffee

```diff
@@ -227,7 +227,8 @@ class TextEditorComponent
     window.addEventListener 'resize', @requestHeightAndWidthMeasurement
 
     @listenForIMEEvents()
-    @trackSelectionClipboard() if process.platform is 'linux'
+    if process.platform is 'linux'
```

Comment:
I checked for the `editor.selectionClipboard` setting here, but decided that it wouldn't be necessary. The line was getting really long with both conditionals so I put it like this, and I forgot to revert the change when I removed it again.

Want me to change it back?